codegen-server/src/test/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ConstrainedNumberGeneratorTest.kt

```diff
@@ -116,6 +121,8 @@ class ConstrainedNumberGeneratorTest {
             listOf(
                 Triple("integer", "ConstrainedInteger", "i32"),
                 Triple("short", "ConstrainedShort", "i16"),
+                Triple("long", "ConstrainedLong", "i64"),
+                Triple("byte", "ConstrainedByte", "i8"),
```

Comment:
Nit: could you order these from `i8` to `i64`?

Reply:
I had an internal bet going on about receiving a comment on this 😛

Reply:
Sorry :grimacing: